inlong-sort/sort-core/src/test/java/org/apache/inlong/sort/parser/ElasticsearchSqlParseTest.java

```diff
@@ -59,7 +58,7 @@ private MySqlExtractNode buildMysqlExtractNode() {
             true, null);
     }
 
-    private ElasticsearchLoadNode buildElasticsearchLoadNode() {
+    ElasticsearchLoadNode buildElasticsearchLoadNode(int i) {
```

Comment:
The parameter `i` is not understood easily.

Reply:
OK, I will modify it. it means version number